starport/pkg/gomodulepath/gomodulepath.go

```diff
@@ -56,16 +51,16 @@ func Parse(rawpath, appPath string) (Path, error) {
 }
 
 // Find search the Go module in the current and parent paths until finding it.
-func Find(path string) (Path, error) {
+func Find(path string) (Path, string, error) {
```

Comment:
still I don't understand why we return the path here that is already given as an argument to the func. the caller of Find already knows what the path is, we don't need to return it here.

Reply:
```suggestion
func Find(path string) (parsed Path, appPath string, err error) {
```